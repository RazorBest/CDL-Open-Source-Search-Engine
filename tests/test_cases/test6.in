no && ((Many&& Linux &&kernel && developers && were && unhappy)||(mind &&to &&the &&youths))