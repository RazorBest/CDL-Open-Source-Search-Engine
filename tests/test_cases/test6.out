example_docs/doc01.txt
example_docs/doc03.txt